kategory/src/main/kotlin/kategory/free/Free.kt

```diff
@@ -6,6 +6,9 @@ typealias FreeF<S> = HK<Free.F, S>
 fun <S, A> FreeKind<S, A>.ev(): Free<S, A> =
         this as Free<S, A>
 
+fun <M, S, A> FreeKind<S, A>.foldMapM(f: FunctionK<S, M>, MM: Monad<M>): HK<M, A> =
+        (this as Free<S, A>).foldMap(f, MM)
```

Comment:
It does collide.

The ev() jump is just a perf hit for the sake of it, I've been inlining it for now.